### ISSUE TITLE
NodeHasPkgCount.java:33-37: This matcher should also...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#285]
The puzzle `241-fffd0470` from #241 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/2bea7533e47134926a99f915a47afae0cd733a48/src/test/java/com/artipie/rpm/hm/NodeHasPkgCount.java#L33-L37

The puzzle was created by olenagerasimova on 18-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8), @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) please, pay attention to this issue

**Comment by olenagerasimova** [comment]
@0crat in

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@773dc7f8-970d-444a-b02a-c8dba678f128, thread: pool-94-thread-1
java.lang.Thread[-2] java.lang.OutOfMemoryError: unable to create new native thread
```

[0.53.15](https://github.com/zerocracy/farm): Issue: artipie/rpm-adapter#285, Comment: 647902715

**Comment by 0crat** [comment]
The job [#285](https://github.com/artipie/rpm-adapter/issues/285) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/00cd0daa-045c-4447-9cf2-58bd9b91288d); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/285#issuecomment-648075265))

@victornoel The user @victornoel[/z](https://www.0crat.com/u/victornoel) resigned from [#285](https://github.com/artipie/rpm-adapter/issues/285), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/285#issuecomment-648075265))

@victornoel Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
The job [#285](https://github.com/artipie/rpm-adapter/issues/285) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/1110b541-69d4-4c43-8371-07710d846f3c); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@0580059e-ceba-491e-980a-2a75bcaf1058, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Week 27' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.53.15](https://github.com/zerocracy/farm): CID: [55614698-37f9-4422-9821-a93adffb94d1](https://www.0crat.com/footprint/CT2E6TK9B/55614698-37f9-4422-9821-a93adffb94d1), Type: "Job milestoned"

**Comment by 0pdd** [comment]
The puzzle `241-fffd0470` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#307](https://github.com/artipie/rpm-adapter/issues/307) is still not solved.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#311](https://github.com/artipie/rpm-adapter/issues/311) are still not solved; solved: [#307](https://github.com/artipie/rpm-adapter/issues/307).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317) are still not solved; solved: [#307](https://github.com/artipie/rpm-adapter/issues/307), [#311](https://github.com/artipie/rpm-adapter/issues/311).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#359](https://github.com/artipie/rpm-adapter/issues/359), [#360](https://github.com/artipie/rpm-adapter/issues/360) are still not solved; solved: [#307](https://github.com/artipie/rpm-adapter/issues/307), [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#360](https://github.com/artipie/rpm-adapter/issues/360) are still not solved; solved: [#307](https://github.com/artipie/rpm-adapter/issues/307), [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317), [#359](https://github.com/artipie/rpm-adapter/issues/359).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#360](https://github.com/artipie/rpm-adapter/issues/360) is still not solved; solved: [#307](https://github.com/artipie/rpm-adapter/issues/307), [#310](https://github.com/artipie/rpm-adapter/issues/310), [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317), [#359](https://github.com/artipie/rpm-adapter/issues/359).
